Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent any
+ 
+    stages {
+        stage('build') {
```

Comment:
extreme nitpick but we should follow the same convention for all stage names. I like Pascal case